<launch>

  <!-- Launch grap_test_rviz.launch to visualize this test -->

  <!-- Debug -->  
  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value="" />
  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />
<!-- 
  <node name="right_gripper_urdf_publisher" pkg="baxter_examples" type="send_urdf_fragment.py" args="-f $(find control_main)/config/right_end_effector.urdf.xacro -l right_hand -j right_gripper_base" output="screen" />
 -->

  <!-- Verbose mode -->
  <arg name="verbose" default="true" />
  <arg unless="$(arg verbose)" name="command_args" value="" />
  <arg     if="$(arg verbose)" name="command_args" value="--verbose" />

  <!-- Load the URDF, SRDF and other .yaml configuration files on the param server -->
<!--   <include file="$(find baxter_moveit_config)/launch/planning_context.launch">
    <arg name="load_robot_description" value="true"/>
  </include> -->

  <!-- Publish world frame -->
  <node pkg="tf" type="static_transform_publisher" name="tf_publisher_world" args="0 0 0 0 0 0 /world /base 25" />

  <!-- Which arm to use -->
  <arg name="arm" default="right"/>

  <!-- Start the test -->
  <node name="grasp_generator_test" launch-prefix="$(arg launch_prefix)" pkg="grasp_generator" 
	type="grasp_generator_test" output="screen" args="$(arg command_args)">
    <param name="arm" value="$(arg arm)"/>
    <rosparam command="load" file="$(find grasp_generator)/config/baxter_grasp_data.yaml"/>    
  </node>

</launch>
